Packs/LinuxEventsCollection/ReleaseNotes/1_0_10.md

```diff
@@ -0,0 +1,7 @@
+
+#### Parsing Rules
+
+##### Linux Events Collection Parsing Rule
+
+- Improved implementation of the existing parsing rule. 
+- Added support for [RFC 3339](https://datatracker.ietf.org/doc/html/rfc3339) compatible like timestamps, such as *`2024-10-28T14:30:55+0300`*, *`2024-10-28T14:30:55.123-03:00`* and *`2024-10-28T14:30:55.123Z`*, on top of the already supported [RFC 3164](https://datatracker.ietf.org/doc/html/rfc3164) compatible timestamps.
```

Comment:
Recommend:
Added support for [RFC 3339](https://datatracker.ietf.org/doc/html/rfc3339) compatible timestamps, such as *`2024-10-28T14:30:55+0300`*, *`2024-10-28T14:30:55.123-03:00`* and *`2024-10-28T14:30:55.123Z`*, in addition to the already supported [RFC 3164](https://datatracker.ietf.org/doc/html/rfc3164) compatible timestamps.

Reply:
Fixed